bot.py

```diff
@@ -431,33 +433,102 @@ def main_color(self):
         sent_emoji = self.config.get('sent_emoji', '✅')
+        blocked_emoji = self.config.get('blocked_emoji', '🚫')
+
+        if sent_emoji not in UNICODE_EMOJI:
+            try:
+                sent_emoji = await converter.convert(
+                    ctx, sent_emoji.strip(':')
+                )
+            except commands.BadArgument:
+                logger.warning(info(f'Sent Emoji ({sent_emoji}) '
+                                    f'is not a valid emoji.'))
+                del self.config.cache['sent_emoji']
+                await self.config.update()
 
         if blocked_emoji not in UNICODE_EMOJI:
             try:
                 blocked_emoji = await converter.convert(
                     ctx, blocked_emoji.strip(':')
                 )
             except commands.BadArgument:
-                pass
+                logger.warning(info(f'Blocked emoji ({blocked_emoji}) '
+                                    'is not a valid emoji.'))
+                del self.config.cache['blocked_emoji']
+                await self.config.update()
+        return sent_emoji, blocked_emoji
 
-        if sent_emoji not in UNICODE_EMOJI:
+    async def process_modmail(self, message):
+        """Processes messages sent to the bot."""
+        sent_emoji, blocked_emoji = await self.retrieve_emoji()
+
+        account_age = self.config.get('account_age')
+        if account_age is None:
+            account_age = isodate.duration.Duration()
+        else:
             try:
-                sent_emoji = await converter.convert(
-                    ctx, sent_emoji.strip(':')
-                )
-            except commands.BadArgument:
-                pass
+                account_age = isodate.parse_duration(account_age)
+            except isodate.ISO8601Error:
+                logger.warning('The account age limit needs to be a '
+                               'ISO-8601 duration formatted duration string '
+                               f'greater than 0 days, not "%s".', str(account_age))
+                del self.config.cache['account_age']
+                await self.config.update()
+                account_age = isodate.duration.Duration()
 
-        if str(message.author.id) in self.blocked_users:
+        reason = self.blocked_users.get(str(message.author.id))
+        if reason is None:
+            reason = ''
+        try:
+            min_account_age = message.author.created_at + account_age
+        except ValueError as e:
+            logger.warning(e.args[0])
+            del self.config.cache['account_age']
+            await self.config.update()
+            min_account_age = message.author.created_at
+
+        if min_account_age > datetime.utcnow():
+            # user account has not reached the required time
             reaction = blocked_emoji
+            changed = False
+
+            if str(message.author.id) not in self.blocked_users:
+                new_reason = 'System Message: New Account.'
```

Comment:
Reason should be clearer. Suggestion: "System Message: User account age too low, below required {required_age}".

Then use a `startswith` later to check

Reply:
Now it's: `System Message: New Account. Required to wait for 1 year, 7 months and 2 days.`